lib/js-yaml/dumper.js

```diff
@@ -220,16 +221,17 @@ function isPlainSafeFirst(c) {
     && c !== CHAR_RIGHT_SQUARE_BRACKET
     && c !== CHAR_LEFT_CURLY_BRACKET
     && c !== CHAR_RIGHT_CURLY_BRACKET
-    // | “#” | “&” | “*” | “!” | “|” | “>” | “'” | “"”
+    // | “#” | “&” | “*” | “!” | “|” | “=” | “>” | “'”
     && c !== CHAR_SHARP
     && c !== CHAR_AMPERSAND
     && c !== CHAR_ASTERISK
     && c !== CHAR_EXCLAMATION
     && c !== CHAR_VERTICAL_LINE
+    && c !== CHAR_EQUALS
     && c !== CHAR_GREATER_THAN
     && c !== CHAR_SINGLE_QUOTE
+    // | “"” | “%” | “@” | “`”)
```

Comment:
Why comment & `"` location was changed?

Reply:
I moved it so that the previous comment didn't get too long for readability. I can undo this change if you think it's unnecessary.

Reply:
Please, roll back this change, to reduce confusion about PR intent.

Reply:
Done!